packages/node_modules/@ciscospark/plugin-phone/src/call.js

```diff
@@ -8,7 +8,13 @@
 /* global RTCPeerConnection, RTCSessionDescription */
 
 import {SparkPlugin} from '@ciscospark/spark-core';
-import {base64, oneFlight, retry, tap, whileInFlight} from '@ciscospark/common';
+import {
+  base64,
```

Comment:
yea, but only for a very specific use case; that was when converting an arraybuffer into a base64 string to put in an image tag. base64url didn't work for that case, but generally speaking, base64url is preferred.